_posts/2021-09-27-solidity-0.8.8-release-announcement.md

```diff
@@ -101,6 +101,31 @@ inside the constructor but you cannot use them to initialize other
 immutable variables at the point of their declaration.
 
 
+### Import Paths
+
+We added a new option `--include-path` to `solc` that hopefully makes it much easier to compile
+more complicated projects using multiple libraries. You can use it to specify
+directories where source files are looked up when `import` statements are resolved. If you use
+`npm` to install Solidity packages, they are usually placed in a subdirectory below
+`node_modules`. If you use imports of the form `import X from "@packagename/contracts/x.sol";`,
+then invoking the compiler as follows should properly resolve all imports without further work:
+
+```bash
+solc contract.sol --base-path . --include-path node_modules/
+```
```

Comment:
Do we want to mention that base path and include paths are automatically added to allowed paths? "without further work" kinda means that but it's not very explicit.

Reply:
yep, will do.